lib/broccoli/ember-app.js

```diff
@@ -976,17 +977,29 @@ EmberApp.prototype._addAppTests = function(sourceTrees) {
         annotation: 'Funnel (jshint app)'
       }), this._prunedBabelOptions());
 
-
       jshintedTests = new Babel(new Funnel(jshintedTests, {
         srcDir: '/',
         destDir: this.name + '/tests/',
         annotation: 'Funnel (jshint tests)'
       }), this._prunedBabelOptions());
 
-      sourceTrees.push(jshintedApp);
-      sourceTrees.push(jshintedTests);
+      appTestTrees.push(jshintedApp);
+      appTestTrees.push(jshintedTests);
     }
-  }
+
+   appTestTrees.push(this._processedEmberCLITree());
+
+    appTestTrees = mergeTrees(appTestTrees, {
+      overwrite: true,
```

Comment:
It seems that both `jshintedTests`and the actual tests tree (e.g., the first one to be pushed) have `tests/index.html` included in them. This wasn't causing issues before since the first time the trees got merged in that case, overwrite is true.